src/web/calculators/arm/components/ArmCalculator.tsx

```diff
@@ -34,11 +37,37 @@ export default function ArmCalculator(): JSX.Element {
         get.endAngle.toDict(),
         get.iterationLimit
       ),
+    kG: () =>
+      calculateKg(
+        get.motor.stallTorque.mul(get.motor.quantity).mul(get.ratio.asNumber()),
+        get.comLength,
+        get.armMass.mul(get.efficiency / 100),
+        nominalVoltage
+      ),
+    kV: () =>
+      calculateKv(
+        get.motor.freeSpeed.div(get.ratio.asNumber()),
+        new Measurement(1, "rad"),
+        nominalVoltage
+      ),
+    kA: () =>
+      calculateKa(
+        get.motor.stallTorque
+          .mul(get.motor.quantity)
+          .mul(get.ratio.asNumber())
+          .mul(get.efficiency / 100),
+        get.comLength.mul(get.comLength).div(new Measurement(1, "rad")),
```

Comment:
you can use `get.comLength.mul(get.comLength).removeRad()` if you want. doesn't matter to me